# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/rene/cups/setprinter.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -ur cups-1.1.18/berkeley/Makefile cups-1.1.18-fixed/berkeley/Makefile
--- cups-1.1.18/berkeley/Makefile	2002-12-17 19:56:34.000000000 +0100
+++ cups-1.1.18-fixed/berkeley/Makefile	2003-05-07 01:26:05.794298504 +0200
@@ -57,11 +57,11 @@
 
 install:	all
 	$(INSTALL_DIR) $(BINDIR)
-	$(INSTALL_BIN) lpq $(BINDIR)
-	$(INSTALL_BIN) lpr $(BINDIR)
-	$(INSTALL_BIN) lprm $(BINDIR)
+	$(INSTALL_BIN) lpq $(BINDIR)/lpq_cups
+	$(INSTALL_BIN) lpr $(BINDIR)/lpr_cups
+	$(INSTALL_BIN) lprm $(BINDIR)/lprm_cups
 	$(INSTALL_DIR) $(SBINDIR)
-	$(INSTALL_BIN) lpc $(SBINDIR)
+	$(INSTALL_BIN) lpc $(SBINDIR)/lpc_cups
 
 
 #
diff -ur cups-1.1.18/man/Makefile cups-1.1.18-fixed/man/Makefile
--- cups-1.1.18/man/Makefile	2002-12-17 19:59:55.000000000 +0100
+++ cups-1.1.18-fixed/man/Makefile	2003-05-07 01:43:21.878789720 +0200
@@ -70,26 +70,30 @@
 install:	all
 	$(INSTALL_DIR) $(MANDIR)/man1
 	for file in $(MAN1); do \
-		$(INSTALL_MAN) $$file $(MANDIR)/man1/`basename $$file man`1; \
+	        inst=`echo $$file | sed -e 's/\\.[0-9a-z]*$$//'`; \
+		$(INSTALL_MAN) $$file $(MANDIR)/man1/$$inst'_cups.1'; \
 	done
-	$(RM) $(MANDIR)/man1/cancel.1
-	$(LN) lp.1 $(MANDIR)/man1/cancel.1
+	$(RM) $(MANDIR)/man1/cancel_cups.1
+	$(LN) lp_cups.1 $(MANDIR)/man1/cancel_cups.1
 	$(INSTALL_DIR) $(PMANDIR)/man3
 	for file in $(MAN3); do \
-		$(INSTALL_MAN) $$file $(PMANDIR)/man3/`basename $$file man`3; \
+	        inst=`echo $$file | sed -e 's/\\.[0-9a-z]*$$//'`; \
+		$(INSTALL_MAN) $$file $(PMANDIR)/man3/$$(inst)'_cups.3'; \
 	done
 	$(INSTALL_DIR) $(MANDIR)/man5
 	for file in $(MAN5); do \
-		$(INSTALL_MAN) $$file $(MANDIR)/man5/`basename $$file man`5; \
+	        inst=`echo $$file | sed -e 's/\\.[0-9a-z]*$$//'`; \
+		$(INSTALL_MAN) $$file $(MANDIR)/man5/$$(inst)'_cups.5'; \
 	done
 	$(INSTALL_DIR) $(AMANDIR)/man$(MAN8EXT)
 	for file in $(MAN8); do \
-		$(INSTALL_MAN) $$file $(AMANDIR)/man$(MAN8EXT)/`basename $$file man`$(MAN8EXT); \
+	        inst=`echo $$file | sed -e 's/\\.[0-9a-z]*$$//'`; \
+		$(INSTALL_MAN) $$file $(AMANDIR)/man$(MAN8EXT)/$$(inst)'_cups'.$(MAN8EXT); \
 	done
-	$(RM) $(AMANDIR)/man$(MAN8EXT)/reject.$(MAN8EXT)
-	$(LN) accept.$(MAN8EXT) $(AMANDIR)/man$(MAN8EXT)/reject.$(MAN8EXT)
-	$(RM) $(AMANDIR)/man$(MAN8EXT)/disable.$(MAN8EXT)
-	$(LN) enable.$(MAN8EXT) $(AMANDIR)/man$(MAN8EXT)/disable.$(MAN8EXT)
+	$(RM) $(AMANDIR)/man$(MAN8EXT)/reject_cups.$(MAN8EXT)
+	$(LN) accept_cups.$(MAN8EXT) $(AMANDIR)/man$(MAN8EXT)/reject_cups.$(MAN8EXT)
+	$(RM) $(AMANDIR)/man$(MAN8EXT)/disable_cups.$(MAN8EXT)
+	$(LN) enable_cups.$(MAN8EXT) $(AMANDIR)/man$(MAN8EXT)/disable_cups.$(MAN8EXT)
 	$(INSTALL_DIR) $(MANDIR)/cat1
 	for file in $(CAT1); do \
 		$(INSTALL_MAN) $$file $(MANDIR)/cat1; \
diff -ur cups-1.1.18/systemv/Makefile cups-1.1.18-fixed/systemv/Makefile
--- cups-1.1.18/systemv/Makefile	2002-12-17 20:00:19.000000000 +0100
+++ cups-1.1.18-fixed/systemv/Makefile	2003-05-07 01:39:48.220270752 +0200
@@ -60,25 +60,22 @@
 
 install:	all
 	$(INSTALL_DIR) $(SBINDIR)
-	$(INSTALL_BIN) accept $(SBINDIR)
-	$(RM) $(SBINDIR)/reject
-	$(LN) accept $(SBINDIR)/reject
+	$(INSTALL_BIN) accept $(SBINDIR)/accept_cups
+	$(LN) accept $(SBINDIR)/reject_cups
 	$(INSTALL_BIN) cupsaddsmb $(SBINDIR)
-	$(INSTALL_BIN) lpadmin $(SBINDIR)
-	$(INSTALL_BIN) lpinfo $(SBINDIR)
-	$(INSTALL_BIN) lpmove $(SBINDIR)
+	$(INSTALL_BIN) lpadmin $(SBINDIR)/lpadmin_cups
+	$(INSTALL_BIN) lpinfo $(SBINDIR)/lpinfo_cups
+	$(INSTALL_BIN) lpmove $(SBINDIR)/lpremove_cups
 	$(INSTALL_DIR) $(BINDIR)
-	$(INSTALL_BIN) cancel $(BINDIR)
+	$(INSTALL_BIN) cancel $(BINDIR)/cancel_cups
 	$(INSTALL_BIN) cupstestppd $(SBINDIR)
-	$(RM) $(BINDIR)/disable
-	$(LN) ../sbin/accept $(BINDIR)/disable
-	$(RM) $(BINDIR)/enable
-	$(LN) ../sbin/accept $(BINDIR)/enable
-	$(INSTALL_BIN) lp $(BINDIR)
-	$(INSTALL_BIN) lpoptions $(BINDIR)
-	$(INSTALL_BIN) lpstat $(BINDIR)
-	$(INSTALL_BIN) lppasswd $(BINDIR)
-	-$(INSTALL_BIN) -m 4755 -o $(CUPS_USER) -g $(CUPS_GROUP) lppasswd $(BINDIR)
+	$(LN) ../sbin/accept $(BINDIR)/disable_cups
+	$(LN) ../sbin/accept $(BINDIR)/enable_cups
+	$(INSTALL_BIN) lp $(BINDIR)/lp_cups
+	$(INSTALL_BIN) lpoptions $(BINDIR)/lpoptions_cups
+	$(INSTALL_BIN) lpstat $(BINDIR)/lpstat_cups
+	$(INSTALL_BIN) lppasswd $(BINDIR)/lppasswd_cups
+	-$(INSTALL_BIN) -m 4755 -o $(CUPS_USER) -g $(CUPS_GROUP) lppasswd $(BINDIR)/lppasswd_cups
 
 
 #
